modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -138,49 +130,115 @@ The ownCloud windows network drive listener utilizes the SMB notification featur
+* optionally, a sub-folder of the share you want to connect to.
 
 [CAUTION]
-.Treat all the parameters as being case-sensitive.
 ====
-Although some parts of the app might work properly regardless of casing, other parts might have problems if the case is not respected.
+_Treat all the parameters as being case-sensitive._ Although some parts of the app might work properly regardless of casing, other parts might have problems if the case is not respected.
+====
+
+[IMPORTANT]
+====
+. When using *Kerberos* authentication, read the following documentation first:
+
+.. xref:enterprise/authentication/kerberos.adoc[Kerberos Authentication]. +
+This documentation is a necessary prerequisite to set up Kerberos for use with ownCloud.
+
+.. xref:special-settings-when-adding-a-kerberos-mountpoint[Kerberos Mountpoint]. +
+There are additional settings required when adding a mount point using the Kerberos authentication.
+
+. When using *Credentials hardcoded in config file* authentication, read the following documentation first:
+
+.. xref:enterprise/authentication/enterprise_only_auth.adoc#notes-for-credentials-hardcoded-in-config-file[Notes for Credentials Hardcoded in Config File]. +
+This documentation is a necessary prerequisite to set up this authentication type.
+
+.. xref:special-settings-when-using-credentials-hardcoded-in-config-file[Special Settings when using 'Credentials hardcoded in config file']. +
+There are special settings required when adding a mount point using this authentication type.
 ====
+--
 
-.Follow this procedure to create a new mount point based on WND
+Follow this procedure to create a new mount point based on WND::
++
+--
 . Enter the ownCloud mount point for your new WND share. This _must not_ be an existing folder.
+
 . Select your authentication method. See xref:enterprise/external_storage/enterprise_only_auth.adoc[Enterprise-Only Authentication Options] for complete information on the available authentication methods.
-+
-.WND mountpoint and authorization credentials
-image::enterprise/external_storage/windows_network_drive/wnd-2.png[WND mountpoint and authorization credentials, width=500]
+
 . Enter the address of the server that contains the WND share.
-. The share name provided by Windows or Samba.
-. The root folder of the share. This is can be a subfolder name, or the `$user` variable for the user's home directory. Note that the LDAP `Internal Username Attribute` must be set to the `samaccountname` for either the share or the root to work, and the user's home directory needs to match the `samaccountname`.
+
+. Use the share name provided by Windows or Samba.
+
+. Optionally, the root folder of the share. This is can be a subfolder name, or the `$user` variable when assigning the user's home directory. Note that the LDAP `Internal Username Attribute` must be set to the `samaccountname` for either the share or the root to work, and the user's home directory needs to match the `samaccountname`.
```

Comment:
```suggestion
. Optionally, the root folder of the share. This can be a subfolder name, or the `$user` variable when assigning the user's home directory. Note that the LDAP `Internal Username Attribute` must be set to the `samaccountname` for either the share or the root to work, and the user's home directory needs to match the `samaccountname`.
```